Taarafo.Core/Services/Foundations/PostImpressions/PostImpressionService.Validations.cs

```diff
@@ -40,6 +40,30 @@ private static void ValidatePostImpressionIsNotNull(PostImpression postImpressio
             }
         }
 
+        private void ValidatePostImpressionIds(Guid postId, Guid profileId)
```

Comment:
Can you change this to ValidatePostImpressionOnRemove and make it work in the same way as ValidatePostImpressionOnAdd.

The way you have it here, it only evaluates 1 before you move on to the next one.
If you use the `Validate(...)` then you will have the error details of both in the validation summary

 Validate(
    (Rule: IsInvalid(postId), Parameter: nameof(postId)),
    (Rule: IsInvalid(profileId), Parameter: nameof(profileId)));

Reply:
Done